internal/rukpak/source/configmaps.go

```diff
@@ -0,0 +1,85 @@
+package source
+
+import (
```

Comment:
We can have just the `internal/rukpak/source/image.go` for now, so that we figure out the unpacker for just one source for now. All the other implementations in `internal/rukpak/source/` can be removed for now